mulighetsrommet-tiltakshistorikk/src/main/kotlin/no/nav/mulighetsrommet/tiltakshistorikk/repositories/DeltakerRepository.kt

```diff
@@ -158,6 +167,7 @@ class DeltakerRepository(private val db: Database) {
                     gruppetiltak.arrangor_organisasjonsnummer
                 from komet_deltaker deltaker join gruppetiltak on deltaker.gjennomforing_id = gruppetiltak.id
                 where deltaker.person_ident = any(:identer)
+                $maxAgeCondition
                 order by deltaker.start_dato desc nulls last;
```

Comment:
Kan evt skrives om til å gjøre null-sjekken i db-spørringen, slik vi har begynt å gjøre en del andre steder. 

I dette tilfellet utgjør det kanskje ikke så stor forskjell, men ved å følge dette mønsteret minsker vi potensielt sjansen for sql-injections og evt. logging av personopplysninger ved at alle inputs sendes med som parametere til postgres i stedet for å embeddes i selve sql'en.

```
...
where deltaker.person_ident = any(:identer)
    and (:years is null or age(coalesce(deltaker.slutt_dato::timestamp, deltaker.registrert_dato)) < INTERVAL ':years years')
...
```

Reply:
Kan fikse